### DISCUSSION TITLE
Using Livewire Pest Helper

### DISCUSSION THREAD

**Discussion by ousid** [spatie/laravel-livewire-wizard#4] [Ideas]
Hey @freekmurze 

I just forked the repo to make a PR, but it turns the repo is not receiving any commits ATM.

I have an idea, since you are using `pestphp` as a testing framework. You can add livewire as a helper instead of `Livewire::test()` method.

## Steps to produce:

require the composer package
```bash
composer require pestphp/pest-plugin-livewire
```

instead of the current `ComponentTest` file, do the following:

```php
<?php

use Livewire\Livewire;
use Spatie\LivewireWizard\Tests\TestSupport\Components\MyWizardComponent;
use Spatie\LivewireWizard\Tests\TestSupport\Components\Steps\FirstStepComponent;
use function Pest\Livewire\livewire;

it('can render a wizard component')
    ->livewire(MyWizardComponent::class)
    ->assertSuccessful();
    
it('can render a step component')
    ->livewire(FirstStepComponent::class)
    ->assertSuccessful();
```

I test it out, and it works just fine.

## ScreenShots:

![image](https://user-images.githubusercontent.com/21012933/168495934-6803eb4d-f28b-4bf6-bee9-e5b96e2e4e06.png)

Hope this helps :)

**Comment by freekmurze** [comment]
Thanks for the suggestion. I'm going to leave things as they are for now.

**Reply by ousid** [reply]
If you want me to do a "refactoring" in the future, I'm happy to do it so!
